apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -419,7 +419,7 @@ def _create_prj_conf(self, project_dir, options):
             f.write("\n")
 
     API_SERVER_CRT_LIBS_TOKEN = "<API_SERVER_CRT_LIBS>"
-    ENABLE_CMSIS_TOKEN = "<ENABLE_CMSIS>"
+    CMAKE_ARGS = "<CMAKE_ARGS>"
```

Comment:
I'd like to use  `CMAKE_ARGS_TOKEN` as the var name here , with `_TOKEN` at the end, like it was before for the `ENABLE_CMSIS_TOKEN` var.

Reply:
done